src/Stripe.net/Infrastructure/Public/StripeConfiguration.cs

```diff
@@ -85,6 +95,12 @@ public static string ApiKey
         /// </summary>
         public static JsonSerializerSettings SerializerSettings { get; set; } = DefaultSerializerSettings();
 
+        /// <summary>
+        /// Gets or sets the maximum number of times that the library will retry requests that
+        /// appear to have failed due to an intermittent problem.
+        /// </summary>
+        public static int MaxNetworkRetries { get; set; } = 0;
```

Comment:
Man, they let you set defaults for auto-properties now too?! Love how much boilerplate they've stripped out of this language.